src/lib/common/JsonHelper.cpp

```diff
@@ -254,6 +280,17 @@ void JsonHelper::addDate(const std::string& key, long long timestamp)
 
 
 
+/* ****************************************************************************
+*
+* JsonHelper::addBool -
+*/
+void JsonHelper::addBool(const std::string& key, bool b)
```

Comment:
First time we need to add booleans to rendered JSON output in history? :)

The addition is great... just a bit surprised we didn't have it already.

Reply:
Yeah, strange, right?
Guess bools were added in some other way before. Haven't looked into it

Reply:
NTC